less/home-page.less

```diff
@@ -19,12 +22,22 @@
     flex-wrap: wrap;
     width: 100%;
 
+
     img {
       width: 28%;
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
     }
+    @media @phone {
+      .sky, .launch {
```

Comment:
Good work targeting the individule image classes. Remember you can use the not sudo class as well. &:not(:last-child){ display: none; }